doc/lrps.md

```diff
@@ -142,7 +142,8 @@ In all cases, the consumer is given an array of `ActualLRPResponse`:
             "source_id": "some-process-guid",
             "space_id": "some-space-guid",
             "space_name": "some-space-name"
-        }
+        },
+        "availability_zone": "some-zone"
```

Comment:
It looks like there's no entry for the new field under the "ActualLRP Identifiers" section. Probably just need to add a small amount of information after line 208.